core/runtime/wasm_edge/module_factory_impl.cpp

```diff
@@ -52,7 +52,7 @@ namespace kagome::runtime::wasm_edge {
 
   static const auto kMemoryName = WasmEdge_StringCreateByCString("memory");
 
-  class WasmEdgeErrCategory final : public std::error_category {
+  static const class final : public std::error_category {
```

Comment:
Something went wrong and class name was lost.
```c++
class WasmEdgeErrCategory final : ... { ... };
static const wasm_edge_err_category;
```

Reply:
Cause is is in-place declaration+definition. Nothing wrong, just anonymous class